packages/ckeditor5-ui/theme/components/spinner/spinner.css

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+:root {
+	--ck-toolbar-spinner-size: 18px;
+}
+
+.ck.ck-spinner-container {
+	display: block;
+	position: relative;
```

Comment:
I know this is weird our system works like this since there's only one theme and we never went on from that place but can you leave the structure (display, position) here and move the presentation (border, width, margin) to ckeditor5-theme-lark? Just in case one day we change our minds.

Reply:
And no... there's no clear guide on this 😃